External/FEXCore/Source/Interface/Core/GdbServer.cpp

```diff
@@ -870,8 +870,13 @@ GdbServer::HandledPacketType GdbServer::handleV(const std::string& packet) {
 
     const auto F       = [](int result) { return fmt::format("F{:x}", result); };
     const auto F_error = [] { return fmt::format("F-1,{:x}", errno); };
-    const auto F_data  = [](int result, const std::string& data) {
-        return fmt::format("F{:x};{}", result, data);
+    const auto F_data  = [](int result, const uint8_t *data, size_t size) {
+      // Binary encoded data is raw appended to the end
+      std::string String = fmt::format("F{:x};", result);
```

Comment:
Why didn't the previous code work? Wouldn't at least `fmt::format("F{:#x};", result) + data` have been equivalent?

Reply:
You're right. Resolved.